hodor

```diff
@@ -44,21 +50,39 @@ class Hodor
     conf = YAML.load(ERB.new(File.read('.hodorfile')).result)
     @containers = conf['containers']
     @tasks = conf['tasks']
+    @hostmanager = conf['host-manager'] || "boot2docker"
+    if @hostmanager == "boot2docker"
+      @host = "boot2docker-vm"
+    elsif @hostmanager == "docker-machine"
+      @host = conf['host'] || ENV['DOCKER_MACHINE_NAME'] || 'default'
+    end
   end
 
   def non_linux_init
-    # Check if boot2dockr is started
+    # Check if hostmanager is started
     # if not display error
-    status = `boot2docker status 2>&1`.strip!
+    if @hostmanager == "boot2docker"
+      status = `boot2docker status 2>&1`.strip!
+    elsif @hostmanager == "docker-machine"
+      status = `docker-machine status #{@host} 2>&1`.downcase.strip!
+    end
     if status != "running"
-      abort("Seems like boot2docker was not started, try to run 'boot2docker init' and then 'boot2docker up'")
+      abort("Seems like #{@hostmanager} was not started, try to run '#{@hostmanager} init' and then '#{@hostmanager} up'")
```

Comment:
not sure if those instructions are relevant for docker-machine? Maybe more generic message?